deployments/k8s/helm/signalfx-agent/templates/clusterrole.yaml

```diff
@@ -71,6 +71,12 @@ rules:
   - get
   - list
   - watch
+- nonResourceURLs:
+  - '*'
```

Comment:
We should limit this to just `/metrics`.

Reply:
I was under the impression that, that path is configurable. Isn't it? 🤔

Reply:
Configurable in Kubernetes or in our monitor?

Reply:
I made the path configurable in the monitor thinking that it's configurable in k8s.

Reply:
It might be but I think /metrics is a pretty standard endpoint for prometheus metrics. I think it's better to limit it to /metrics and if it's wrong for their environment they can change the role.

Reply:
ok